canonical-web-and-design/ubuntu.com
# Add image_template to the /shared/pricing includes

## Done

Replaced images with the image_template module, mostly the aubergine ticks

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/pricing/infra to see most of them
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the images are still there (compare to ubuntu.com)
